pkgs/development/python-modules/scikit-survival/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchPypi
+, git
+, setuptools_scm
+, cython
+, scikitlearn
+, cvxpy
+, cvxopt
+, numexpr
+, osqp
+, pandas
+, scipy
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "scikit-survival";
+  version = "0.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0q47ifqrjyviszxz68hcgk10wcajnp9cinvl45h5zw6a478y7j15";
+  };
+
+  # Works with pandas >= 1.0.1
+  # waiting for upstream to merge and release new version
+  postPatch = ''
+    substituteInPlace requirements/prod.txt \
+      --replace "pandas >=0.21,<0.26" "pandas >=1.0.1"
+  '';
+  
+  nativeBuildInputs = [
+    git  # required for setuptools_cm
+    setuptools_scm
+    cython
+  ];
+
+  propagatedBuildInputs = [
+    scikitlearn
+    cvxpy
+    cvxopt
+    numexpr
+    osqp
+    pandas
+    scipy
+  ];
+
+  checkInputs = [
+    pytest
+  ];
+
+  checkPhase = "
+    pytest tests/
+  ";
+
+  meta = with lib; {
+    description = "Survival analysis built on top of scikit-learn";
+    homepage = "https://github.com/sebp/scikit-survival";
+    license = licenses.gpl3;
```

Comment:
`licenses.gpl3` is deprecated, consider using `licenses.gpl3Only` or `licenses.gpl3Plus`.